src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -98,6 +99,12 @@ public partial class DynamoView : Window, IDisposable
 
         internal ShortcutToolbar ShortcutBar { get { return shortcutBar; } }
 
+        internal PreferencesView PreferencesWindow {
+            get { return preferencesWindow; }
+        }
+
+        internal event Action OnPreferencesWindowChanged;
```

Comment:
The library extension didn't have access to the new PreferencesView object created. So, this event was added to be called after a new preferences window is opened.